### PR TITLE
django v1.8.x - base/english version update

### PR DESCRIPTION
**Pull request by soutys**
Major (DB migrations, steps, URL-s) and minor (bindings, typos, indentation, doc-tests) fixes.

To be merged/synced with https://github.com/Natim/django-carrots/pull/14 !

### CODE REVIEW

**Comment by plocharz-9livesdata** [comment]
Thanks for an update. I'll look at this probably during the weekend.

File: source/django_models.rst [hunk 1]
```diff
@@ -22,21 +22,32 @@ In the file ``polls/models.py`` type::
         choice_text = models.CharField(max_length=200)
         votes = models.IntegerField(default=0)
 
-By adding new models we have changed the database schema. We need to run ``syncdb`` again so that 
+By adding new models we have changed the database schema. In other words we have to *migrate database schema* from one to another. We need to run ``makemigrations`` and then ``migrate`` so that 
 new models can appear in the database.
 
 .. warning::
-    After executing the ``syncdb``, you can not add new fields to the model. You can only add new models. There are ways to avoid it, but … it is a totally different story
+    After executing the ``makemigrations`` and ``migrate``, you cannot add new fields to the model. You can only add new models. There are ways to avoid it, but... it is a totally different story.
```
Comment (qrees) [inline source/django_models.rst]: Is this true?

**Comment by qrees** [comment]
There is something strange with branches, so I'll create another pull request for these changes to be merged with master branch.

**Comment by qrees** [comment]
There is now pull request #51

File: source/django_models.rst (hunk 1 above)
Comment (soutys) [inline source/django_models.rst]: With v1.8.2's admin panel w have:
```
OperationalError at /admin/polls/poll/
no such column: polls_poll.<new_column>
```
where `<new_column>` is th one added without `makemigrations`+`migrate`. Even when self-checks are OK:
```System check identified no issues (0 silenced).```

Comment (qrees) [inline source/django_models.rst]: Doesn't makemigrations and migrate create also new columns?

**Pull request merged**